### PR TITLE
Trip planner fixes and improvements

### LINKED ISSUE #4692: Trip planner doesn't work
# Bug

**Describe the bug**
`TypeError: coordinates.mapProjCoordinate is undefined`

**To Reproduce**
1. Go to 'TripPlanner' panel
2. Click on the map
3. See error

**Expected behavior**
Waypoint added

**Screenshots**
none

**Additional context**
Need to investigate further. Found in examples.

### PR DESCRIPTION
**Pull request by jmacura**
## Description

- Fixes wrong format of clicked coordinates causing errors (#4692)
- Fixes missing panel header
- Improves panel features with an option to select transportation mode ("profile")

Ideas for translations to further langs welcomed, @FilipLeitner ;-)

## Related issues or pull requests

resolves #4692 

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [x] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [MIT License]
- [x] I have followed the [guidelines for contributing](https://github.com/hslayers/hslayers-ng/blob/master/CONTRIBUTING.md)
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/hslayers/hslayers-ng/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

File: projects/test-app/src/hslayers-app/hslayers-app.component.ts [hunk 1]
```diff
@@ -509,6 +510,7 @@ export class HslayersAppComponent {
             synchronize: false,
             cluster: true,
             inlineLegend: true,
+            legend: {},
```
Comment (jmacura) [inline projects/test-app/src/hslayers-app/hslayers-app.component.ts:513]: huh, unintentional leak, will remove...

File: projects/hslayers/components/trip-planner/route-profile-selector.component.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<div class="flex-row w-100 m-auto justify-content-center align-items-center" style="display: flex">
+  <p class="m-0 p-0 flex-fill text-primary">{{'TRIP_PLANNER.selectProfile' | translateHs}}:</p>
+  <div ngbDropdown placement="bottom" style="max-width: 50%">
+    <button type="button" ngbDropdownToggle
+      class="btn btn-sm rounded-0 hs-toolbar-button d-flex align-items-center mw-100" aria-haspopup="true"
+      (click)="profilesExpanded = !profilesExpanded"
+      [attr.aria-expanded]="profilesExpanded">
+      <div class="text-truncate" *ngIf="selectedProfile !== undefined">
```
Comment (FilipLeitner) [inline projects/hslayers/components/trip-planner/route-profile-selector.component.html:8]: We could maybe use new control flow as per https://github.com/hslayers/hslayers-ng/issues/4905

**Comment by FilipLeitner** [comment]
Not sure whether I had to set up something in test-app before being able to use the feature but Iam getting 404
when trying to add waypoints
image

**Review by FilipLeitner — APPROVED** [review]

**Comment by FilipLeitner** [comment]
Added quick UI update and transalations
image

File: projects/hslayers/assets/locales/sk.json [hunk 3]
```diff
@@ -1017,7 +1018,20 @@
     "travelRoute": "Trasa výletu",
     "waypointDrawingHint": "Kliknite prosím do mapy pre pridanie bodov trasy",
     "waypointLayer": "Vrstva s bodmi trasy",
-    "waypoints": "Body trasy"
+    "waypoints": "Body trasy",
+    "noRoutablePoint": "Nenašla sa žiadna trasa medzi zadanými bodmi",
+    "profiles": {
+      "cycling-electric": "elektrobicykel",
+      "cycling-mountain": "bicykel hroský",
```
Comment (jmacura) [inline projects/hslayers/assets/locales/sk.json:1025]: typo? :-)

**Comment by jmacura** [comment]
Thanks, I appreciate the UI changes!

**Pull request merged**